### PR TITLE
Improve readme.

### PR DESCRIPTION
**Pull request by Venemo**
I added some further info and thoughts to the readme. Is there anything else that would be useful to mention here?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,11 +21,96 @@ are not returned to the client process address space, and in the case
...
+* Perform cryptographic operations on behalf of clients
+
+Rationale:
+
+* The user's secret keys are never actually exposed to the client application, so
+the user doesn't have to trust the client to handle sensitive data correctly. In fact, if
+you have a plugin that enables the use of a suitable peripheral, the key is never even loaded
+into system memory.
+* Clients don't have to "know about cryptography" or be certified, because all crypto
+operations are done by the daemon.
```
Comment (chriadam) [inline README.md]: are done by plugins loaded by the daemon

**Comment by chriadam** [comment]
One comment, aside from that LGTM, thanks!

File: README.md (hunk 1 above)
Comment (Venemo) [reply README.md]: Will fix.

**Comment by Venemo** [comment]
@chriadam Fixed it for you!

**Comment by chriadam** [comment]
LGTM thanks

**Pull request merged**